src/python/main.cpp

```diff
@@ -1,6 +1,9 @@
 #include <pybind11/iostream.h>
 #include <pybind11/pybind11.h>
 #include <pybind11/stl.h>
+#include <pybind11/complex.h>
+#include <pybind11/functional.h>
+#include <pybind11/chrono.h>
```

Comment:
this one neither I think

Reply:
I get this when I leave it out:

Did you forget to `#include <pybind11/stl.h>`? Or <pybind11/complex.h>,
<pybind11/functional.h>, <pybind11/chrono.h>, etc. Some automatic
conversions are optional and require extra headers to be included
when compiling your pybind11 module.

Reply:
Perhaps there is a nicer workaround?

Reply:
yes, only remove chrono and complex.h

functional is necessary and used.

Reply:
makes sense...